hd_servo/MotorCommand[] motor_commands
bool block
